src/Tools/BuildBoss/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace BuildBoss
+{
+    internal static class Program
+    {
+        internal static int Main(string[] args)
+        {
+            var sourceDir = args[0];
```

Comment:
I hate programs that require arguments that crash if you don't give them any.  Can you add a usage message?